denonavr/denonavr.py

```diff
@@ -1032,6 +1052,14 @@ def _get_status_from_xml_tags(self, root, relevant_tags):
             elif child.tag == "FriendlyName" and self._name is None:
                 self._name = child[0].text
                 relevant_tags.pop(child.tag, None)
+            elif (child.tag == "selectSurround" or child.tag == "SurrMode"):
+                sound_mode_raw = child[0].text.rstrip()
+                self._sound_mode = self.match_sound_mode(sound_mode_raw)
```

Comment:
Could you move the` match_sound_mode` to the `sound_mode` property?
You won't need the `self. _sound_mode_match_warning` attribute then.

Reply:
just did it.